backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -0,0 +1,183 @@
+import { bibiBloxberg } from '@/seeds/users/bibi-bloxberg'
+import { peterLustig } from '@/seeds/users/peter-lustig'
+import { createContribution } from '@/seeds/graphql/mutations'
+import { login } from '@/seeds/graphql/queries'
+import { cleanDB, resetToken, testEnvironment } from '@test/helpers'
+import { GraphQLError } from 'graphql'
+import { User } from '@entity/User'
+import { Contribution } from '@entity/Contribution'
+import { userFactory } from '@/seeds/factory/user'
+import { garrickOllivander } from '@/seeds/users/garrick-ollivander'
+import { stephenHawking } from '@/seeds/users/stephen-hawking'
+
+let mutate: any, query: any, con: any
+let testEnv: any
+
+beforeAll(async () => {
+  testEnv = await testEnvironment()
+  mutate = testEnv.mutate
+  query = testEnv.query
+  con = testEnv.con
+  await cleanDB()
+})
+
+afterAll(async () => {
+  await cleanDB()
+  await con.close()
+})
+
+let user: User
+let creation: Contribution | void
+
+describe('ContributionResolver', () => {
+  describe('createContribution', () => {
+    describe('unauthenticated', () => {
+      it('returns an error', async () => {
+        await expect(
+          mutate({
+            mutation: createContribution,
+            variables: { amount: 100.0, memo: 'Test Contribution', creationDate: 'not-valid' },
+          }),
+        ).resolves.toEqual(
+          expect.objectContaining({
+            errors: [new GraphQLError('401 Unauthorized')],
+          }),
+        )
+      })
+    })
+
+    describe('authenticated with deleted user', () => {
```

Comment:
This is not needed. Done by the user resolver tests, as a deleted user cannot login